app-alternatives/tar/tar-0.ebuild

```diff
@@ -4,7 +4,7 @@
 EAPI=8
 
 ALTERNATIVES=(
-	"gnu:>=app-arch/tar-1.34-r2"
```

Comment:
You could keep this part if you want, but we don't really need it.

Reply:
Sounds good, dropped.